test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2392,27 +2392,59 @@ UCS_TEST_P(test_ucp_sockaddr_protocols_diff_config,
     test_tag_send_recv(4 * UCS_KBYTE, false, false);
 }
 
+UCS_TEST_P(test_ucp_sockaddr_protocols_diff_config,
```

Comment:
maybe just add test variant to test_ucp_sockaddr_protocols_diff_config so it will run all cases with UCX_CM_USE_ALL_DEVICES=y and UCX_CM_USE_ALL_DEVICES=n
for example https://github.com/openucx/ucx/blob/bc156e0fcad5b6c98cddc8cbfaa670d45bf776a5/test/gtest/ucp/test_ucp_sockaddr.cc#L1163

Reply:
Done